src/main/java/com/alistats/discorki/tasks/CheckJustInGameTask.java

```diff
@@ -58,8 +61,8 @@ public void checkJustInGame() {
                     Optional<CurrentGameInfoDto> currentGameInfoDtoOpt = getCurrentGame(s.getId());
                     if (currentGameInfoDtoOpt.isPresent()) {
                         CurrentGameInfoDto currentGameInfoDto = currentGameInfoDtoOpt.get();
-                        if (s.getCurrentMatch() != null) {
-                            if (s.getCurrentMatch().getId() == currentGameInfoDto.getGameId()) {
+                        if (s.getMatchInProgress().isPresent()) {
+                            if (s.getMatchInProgress().get().getId() == currentGameInfoDto.getGameId()) {
```

Comment:
You can probably replace all this logic with a single line, using something like:
```
currentGameInfoDtoObject.map(...::getCurrentGame).map(...::getId).orElse(0); // or null..
```
or something similar

Reply:
Ty for the suggestion.